vitrina/orgs/services.py

```
@@ -451,6 +596,11 @@ def get_coordinators_count(model: Type[Model], object_id: int) -> int:
     ).count()
 
 
+def can_manage_information_system(object: Dataset, user: User) -> bool:
+    subclass = object.subclass
```

No need to check if object.subclass is not a falsy value? If it can be a falsy value, the next line will fail hard.